fn foo {
  self # error: [line 2] Syntax error at "self": Can't use 'self' outside of a type.
}
